src/app/containers/PodcastExternalLinks/index.jsx

```diff
@@ -79,22 +83,20 @@ const PodcastExternalLinks = ({ brandTitle, links }) => {
 
   return (
     <Wrapper role="complementary" aria-labelledby="third-party-links">
-      <StyledSectionLabel
+      <ThirdPartyLinksTitle
         script={script}
         service={service}
-        labelId="third-party-links"
-        bar={false}
-        dir={dir}
+        id="third-party-links"
       >
         {title}
-      </StyledSectionLabel>
+      </ThirdPartyLinksTitle>
       {hasMultipleLinks ? (
         <StyledList role="list">
           {links.map(({ linkText, linkUrl }) => (
             <StyledListItem dir={dir} key={linkText}>
               <Link href={linkUrl} service={service} script={script} dir={dir}>
                 <span role="text">
-                  {linkText}
+                  <span lang={lang}>{linkText}</span>
```

Comment:
What are your thoughts for adding this span here?

Reply:
Hi @rebeccamcginn. @greenc05 advised adding lang separately just for the link text as seen in this example 
![image](https://user-images.githubusercontent.com/2413911/111619822-0ece3900-87f7-11eb-97c8-0b32039c52b9.png)

Reply:
Cool. We don't need to pass down the language from `OnDemandAudioPage`; we can get it from serviceContext in `PodcastExternalLinks`, line 72

Reply:
Wow! Thanks for the heads up. Let me clean it up right away

Reply:
@rebeccamcginn I've updated the PR. Stopped passing `lang` down from parent components and instead used rge `serviceContext`

Reply:
Hey @mukama, is this treatment also needed on [L116](https://github.com/bbc/simorgh/blob/01db1e971ef0c151556484bf7c66b7de21829e29/src/app/containers/PodcastExternalLinks/index.jsx#L116)?

Reply:
I believe so. Nice catch @simonsinclair

Reply:
I've made the update @simonsinclair